include/lsst/afw/image/VisitInfo.h

```diff
@@ -174,6 +174,9 @@ class VisitInfo : public table::io::PersistableFacade<VisitInfo>, public table::
     // get hour angle at the boresight
     geom::Angle getBoresightHourAngle() const;
 
+    // get parallactic angle at the boresight
```

Comment:
Please expand this comment so define the returned angle -- it is the angle from what to what?